openshift/test/ansible/roles/foreman_repositories/tasks/release_repos.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: 'Setup Foreman Repository'
+  yum:
+    name: http://yum.theforeman.org/{{ "releases/" if foreman_repositories_version != 'nightly' else '' }}{{ foreman_repositories_version }}/el{{ ansible_distribution_major_version }}/x86_64/foreman-release.rpm
+    state: present
+  when: foreman_repositories_version != 1.13 or ansible_distribution_major_version != "6"
+
+- name: 'Setup Foreman Repository'
+  yum:
+    name: http://fedorapeople.org/groups/katello/releases/yum/foreman/1.13/el{{ ansible_distribution_major_version }}/x86_64/foreman-release.rpm
+    state: present
+  when: foreman_repositories_version == 1.13 and ansible_distribution_major_version == "6"
```

Comment:
Haha no -- this was copied from the foreman_repositories role that I think still exists with this in it today.